backend/app/tests/utils/test_jsonl_service.py

```
@@ -0,0 +1,431 @@
+        assert stats_before["total_lines"] > 0
+        
+        service.reset_stats()
+        
+        stats_after = service.get_processing_stats()
+        assert stats_after["total_lines"] == 0
+
+    async def test_cache_clearing(self, service, sample_jsonl):
+        """测试缓存清理"""
+        # 添加一些缓存项
+        await service.process_content(sample_jsonl)
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)